Branch-SDK/src/main/java/io/branch/referral/PrefHelper.java

```diff
@@ -735,11 +735,15 @@ public JSONObject getReferringURLQueryParameters()  {
 
         String string = getString(KEY_REFERRING_URL_QUERY_PARAMETERS);
         JSONObject params = new JSONObject();
-        try {
-            params = new JSONObject(string);
-        }
-        catch (JSONException e) {
-            PrefHelper.LogException("Unable to get URL query parameters as string: ", e);
+
+        if (!TextUtils.isEmpty(string) && !PrefHelper.NO_STRING_VALUE.equals(string)) {
+            try {
+                params = new JSONObject(string);
+            } catch (JSONException e) {
+                PrefHelper.Warning("Unable to get URL query parameters as string: " + e);
+            }
+        } else {
+            PrefHelper.Debug("There are no saved referring URL query parameters.");
```

Comment:
This is probably going to be a common message e.g organic opens. Maybe something simpler like "No url parameters found"